### DISCUSSION TITLE
MegaLinter configuration

### DISCUSSION THREAD

**Discussion by nvuillam** [ngoduykhanh/PowerDNS-Admin#1396] [General]
Thanks for using MegaLinter :)

I see lot of failing linters, you should define a .mega-linter.yml file at the root of your repo to select which linter you do not want to run, which ones you want to be not blocking, etc ...

You can see a mini tutorial in the paragraph "Configuration" in this article :)

https://medium.com/@nicolasvuillamy/improve-uniformize-and-secure-your-code-base-with-megalinter-62ebab422c1

**Comment by AzorianMatt** [comment]
@nvuillam thanks for your feedback. I just added this file yesterday and simply haven't had the time to dive in to particular configuration yet. I do plan to trim a lot of fat and customize it much further, it's simply in there right now to get the ball rolling.

If you have some great examples for production already available, I would gladly welcome them for consideration.

Thank you for being a PDA user!

**Comment by nvuillam** [comment]
You can check MegaLinter repo's own config file :)

https://github.com/oxsecurity/megalinter/blob/main/.mega-linter.yml
